web-api/tests/legacy_tenant_migration.rs

```diff
@@ -132,3 +142,163 @@ async fn test_if_the_initializations_works_correct_for_not_setup_legacy_tenants(
     conn.close().await?;
     Ok(())
 }
+
+#[derive(Deserialize)]
+struct PersonalizedDocumentData {
+    id: String,
+}
+
+#[derive(Deserialize)]
+struct SemanticSearchResponse {
+    documents: Vec<PersonalizedDocumentData>,
+}
+
+//FIXME Once the "old" version we test migration against is the version where this
+//      test was added we can simplify it a lot by using the Silo API and the additional
+//      integration test utils added in this and the previous PR.
+#[tokio::test]
+async fn test_full_migration() -> Result<(), Error> {
+    use old_xayn_web_api::{
+        config as old_config,
+        start as start_old,
+        Ingestion as OldIngestion,
+        Personalization as OldPersonalization,
+        ELASTIC_MAPPING,
+    };
+
+    let (pg_config, es_config) = legacy_test_setup().await?;
+    let config = build_test_config_from_parts(&pg_config, &es_config, Table::new());
+
+    let client = reqwest::Client::new();
+
+    let es_mapping: Value = serde_json::from_str(ELASTIC_MAPPING)?;
+    // the old setup didn't setup elastic search itself, nor has the config pub fields
+    send_assert(
+        &client,
+        client
+            .put(format!("{}/{}", es_config.url, es_config.index_name))
+            .json(&es_mapping)
+            .build()?,
+        StatusCode::OK,
+    )
+    .await;
+
+    let args = &[
+        "integration-test",
+        "--bind-to",
+        "127.0.0.1:0",
+        "--config",
+        &format!("inline:{config}"),
+    ];
+
+    dbg!(&args);
```

Comment:
❓ is this a leftover from local testing?

Reply:
yesn't if the test fails we very likely want to have  this and if it doesn't fail the test output capturing will capture it

Reply:
I removed it but doesn't your IDE by default collapse tests which pass?  I mean IDE always displaying all output of all tests no matter weather the test passes or not seems quite bad design IMHO.